NavigationMenu.md

```diff
@@ -27,6 +27,18 @@
 * ## [Message](./guides/V1_message_main.md)
   * [Send a message into a conversation](./guides/V1_wwsg_Spaces.md)
   * [View a message from a conversation](./guides/V1_get_message.md)
+  * [Delete a message](./guides/V1_delete_message.md)
+  * [Events for message deletion (message-deleted)](./guides/V1_wwsg_Webhooks.md)
```

Comment:
Are we going to have a section for events related to each object or do we keep all events in one place @jarusso